aQute.libg/test/aQute/lib/xmldtoparser/ComponentsDTO.java

```diff
@@ -0,0 +1,111 @@
+package aQute.lib.xmldtoparser;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAttribute;
```

Comment:
This package is removed from Java SE after Java 8. So I am not sure we should rely upon it in Bnd. Using it may produce deprecation warnings in Java 9 and 10 and may fail in Java 11.

Reply:
Actually it fails in the Java 9 test build...